content/styles/spotify/Better-Spotify.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Better-Spotify
+info: Cleaner, and better version of the Spotify Web app with Focus Mode to remove distractions.
+author: Abhay Aysola
+license: MIT License
+home: https://github.com/AbhayAysola/userstyles
+install: https://github.com/AbhayAysola/userstyles/raw/main/Better-Spotify.user.styl
+preview: https://user-images.githubusercontent.com/67417765/98918597-d25e9980-24f3-11eb-9da9-0bc3d02e6e91.png
+thumbnail: https://user-images.githubusercontent.com/67417765/98918696-f1f5c200-24f3-11eb-88e8-18fe58a88e6f.png
```

Comment:
I really like the idea of using a different image for thumbnails.

The problem is it's size of 777KB due to image resolution. Would you like me to merge this PR and tweak the image later? I need to do that for a few other userstyles, so I figured I'd also include this one.